components/Feed/index.js

```diff
@@ -61,15 +90,51 @@ const greetingSubscription = gql`
   }
 `
 
+const dateFormat = timeFormat('%A,\n%d.\xa0%B\xa0%Y')
+
+const groupByDate = nest().key(d => dateFormat(new Date(d.meta.publishDate)))
+
 class Feed extends Component {
+  constructor (props) {
+    super(props)
+    this.pagesLoaded = 0
+    this.state = {
+      infiniteScroll: false
+    }
+    this.onScroll = () => {
+      const { infiniteScroll } = this.state
+      const { loadMore, hasMore, maxPages } = this.props
+      const d = document.documentElement
+      const scrollTop = Math.max(window.pageYOffset, d.scrollTop, document.body.scrollTop)
+      const offset = scrollTop + window.innerHeight
+      const height = d.offsetHeight
+      const needsMore = height - offset < (height / 3)
+      if ((infiniteScroll || this.pagesLoaded < maxPages) && hasMore && needsMore) {
```

Comment:
If I understand correctly, `maxPages` is only relevant until the user has activated infinite scroll. I was slightly confused in the beginning about how infinity and maxPages relate to each other. So maybe we could use a slightly more specific name like `maxInitialPages` to make that clearer at first read.

Reply:
Good point